fourkitchens/sous-drupal-distro
# Fix entity browsers for teaser media fields and convert to htabs

### Purpose:
- Fixes modal displays so they aren't opened on page load and have a responsive height.
- Adds `Tags` exposed filter to media type entity browsers
- Converts node tabs to horizontal due to better Claro support
- Hides core 'Promote to frontpage' and `Sticky at top of lists` node options

### Testing:
- Go to the following pages and verify... `...node/add/frontpage`
     - [x] The tabs are no in a horizontal display
     - [x] The Teaser tab has a description that defines the purpose of these fields for the editors.
     - [x] Opening the Teaser Media field expands the modal to the height of the browser window automatically
     - [x] The Teaser Media field doesn't open the media browser by default
     - [x] The the teaser media entity browser includes an exposed filter for `tags`
     - [x] The teaser media browser is limited to image media references

- Go to the following pages and verify... `...node/add/page`
     - [x] The tabs are no in a horizontal display
     - [x] The Teaser tab has a description that defines the purpose of these fields for the editors.
     - [x] Opening the Teaser Media field expands the modal to the height of the browser window automatically
     - [x] The Teaser Media field doesn't open the media browser by default
     - [x] The the teaser media entity browser includes an exposed filter for `tags`
     - [x] The teaser media browser is limited to image media references

Horizontal vs Vertical tabs in Claro:
<img width="877" alt="Screen Shot 2021-03-14 at 6 10 45 PM" src="https://user-images.githubusercontent.com/369018/111090964-789cc900-84ff-11eb-905b-e1c61529960f.png">
<img width="913" alt="Screen Shot 2021-03-14 at 6 10 36 PM" src="https://user-images.githubusercontent.com/369018/111090972-7dfa1380-84ff-11eb-82a6-15b414f892d1.png">
